libs/category/state/src/facades/category-facade.interface.ts

```diff
@@ -26,7 +27,16 @@ export interface DaffCategoryFacadeInterface<
   /**
    * The page configuration state for the selected category.
    */
-  pageConfigurationState$: Observable<DaffCategoryPageConfigurationState>;
+  pageConfigurationState$: Observable<DaffStatefulCategoryPageConfigurationState>;
+  /**
+   * The current loading state of the selected category page.
+   */
+  // TODO: fix this horrible overuse of "state"
+  pageLoadingState$: Observable<DaffStatefulCategoryPageConfigurationState['daffState']>;
+  /**
+   * Whether the selected category is in a mutating state.
+   */
+  isPageMutating$: Observable<boolean>;
```

Comment:
I think there should be an `isPageResolving` field too

Reply:
added